pages/tool_assembly/NO_TSD_assembly.md

```diff
@@ -0,0 +1,59 @@
+
+The sensitive data tools assembly provides [Nettskjema](https://nettskjema.no) as a solution for designing and managing data collections using online forms and surveys. This is a secure and GDPR-compliant service. It can be accessed through the  UiO's web pages and it is used through a web browser. Submissions from a Nettskjema questionnaire can be delivered securely (fully encrypted) to your project area within TSD. 
+TSD-users are granted access to Nettskjema through [IDporten or FEIDE](https://www.uio.no/tjenester/it/adm-app/nettskjema/mer-om/eksterne-brukere). When the Nettskjema form is complete, you can upload it on TSD following [these instructions](https://www.uio.no/tjenester/it/adm-app/nettskjema/hjelp/koble-skjema-til-tsd.html). After verification, the form can be used for collecting sensitive data. Note that further processing and analysis of the results should be conducted within TSD. If exporting data is necessary, the files should be properly [de-identified or anonymised](sensitive_data.html#how-can-you-de-identify-your-data). 
+
+
+
+For [Processing](processing) and [Analysing](analysing) your data,  you can use singularity containers and [individual tools on the HPC cluster](https://www.uio.no/english/services/it/research/sensitive-data/use-tsd/hpc/software/). 
+The computing services provided through TSD include an Illumina DRAGEN (Dynamic Read Analysis for GENomics) node, which can be used to speed up genomic analysis of sequencing data. Dragen is a dedicated resource, if you want to run jobs on DRAGEN please send an email to [tsd-drift@usit.uio.no](mailto:tsd-drift@usit.uio.no).
+ 
+One solution for permanent archiving and sharing of personally identifiable genetic and phenotypic datasets resulting from biomedical research data is to deposit them to the [European Genome-phenome Archive (EGA)](https://ega-archive.org/). The EGA applies a controlled access model. There can be limitations, e.g. given consents, for your datasets which prevents them from leaving your jurisdiction or being archived in general. This will be partly addressed in the future by federated EGA services with nodes operating from one country or institution under one specific jurisdiction. This model will enable discovery of publicly shareable metadata about studies/datasets archived at the federated EGA nodes through the Central EGA, while the remaining data is stored in a local solution. These nodes will offer the same APIs and interfaces as the Central EGA and provide independent data distribution to users.
+The Norwegian federated EGA (NFEGA) will be accessible through [ELIXIR AAI](https://elixir-europe.org/services/compute/aai), compatible with [FEIDE](https://www.feide.no/).
+
+
+## Where can I find training materials, documentations and events about the NeLS data management tool assembly?
```

Comment:
replace NeLS with TSD in the paragraph title.

Reply:
Thanks for spotting this